README.md

```diff
@@ -1,4 +1,13 @@
 # petclinic-order
 
+## Liquibase manual configuration
+You need to execute next SQL queries to init PostgreSQL DB:
+- CREATE USER pc_admin;
```

Comment:
you use specific password below... where did you mention that password when creating user?